reference/sql/statements/flashback-table.md

```diff
@@ -0,0 +1,84 @@
+---
+title: FLASHBACK TABLE
+summary: Learn how to recover tables using the `FLASHBACK TABLE` statement.
+category: reference
+---
+
+# FLASHBACK TABLE
+
+The `FLASHBACK TABLE` statement is introduced in TiDB 4.0. You can use the `FLASHBACK TABLE` statement to restore the tables and data dropped by `DROP` or `TRUNCATE` within the Garbage Collection (GC) lifetime.
+
+Use the following command to query the TiDB cluster's `tikv_gc_safe_point` and `tikv_gc_life_time`. As long as the table is dropped by `DROP` or `TRUNCATE` after the `tikv_gc_safe_point` time, you can restored the table using the `FLASHBACK TABLE` statement.
+
+{{< copyable "sql" >}}
+
+  ```sql
+  select * from mysql.tidb where variable_name in ('tikv_gc_safe_point','tikv_gc_life_time');
+  ```
+
+## Syntax
+
+{{< copyable "sql" >}}
+
+```sql
+FLASHBACK TABLE table_name [TO other_table_name]
+```
+
+## Notes
+
+If a table is dropped and the GC lifetime has passed, you can no longer use the `FLASHBACK TABLE` statement to recover the dropped data. Otherwise, an error will be returned, which is similar to `Can't find dropped / truncated table 't' in GC safe point 2020-03-16 16:34:52 +0800 CST`.
```

Comment:
```suggestion
If a table is dropped and the GC lifetime has passed, you can no longer use the `FLASHBACK TABLE` statement to recover the dropped data. Otherwise, an error like `Can't find dropped / truncated table 't' in GC safe point 2020-03-16 16:34:52 +0800 CST` will be returned.
```